Signal.xcodeproj/project.pbxproj

```diff
@@ -55,6 +55,12 @@
 		3421981C21061D2E00C57195 /* ByteParserTest.swift in Sources */ = {isa = PBXBuildFile; fileRef = 3421981B21061D2E00C57195 /* ByteParserTest.swift */; };
 		34277A5E20751BDC006049F2 /* OWSQuotedMessageView.m in Sources */ = {isa = PBXBuildFile; fileRef = 34277A5C20751BDC006049F2 /* OWSQuotedMessageView.m */; };
 		3427C64320F500E000EEC730 /* OWSMessageTimerView.m in Sources */ = {isa = PBXBuildFile; fileRef = 3427C64220F500DF00EEC730 /* OWSMessageTimerView.m */; };
+		342950822124C9750000B063 /* OWSTextField.m in Sources */ = {isa = PBXBuildFile; fileRef = 3429507E2124C9740000B063 /* OWSTextField.m */; };
```

Comment:
You can't subclass Swift classes in Obj-C, so I made all of these Obj-C since we sometimes subclass them.

Reply:
You can subclass Swift classes in objc, you just have to make sure the Swift classes are marked @objc (and thus inherit from NSObject)

Reply:
Oh nice.